### PR TITLE
BGDIINF_SB-1503 changing setting for SECURE_PROXY_SSL_HEADER

### PR DESCRIPTION
**Pull request by boecklic**
Django doesn't correctly determine if the original request was made using SSL with our Stack. Therefore we need to adapt the settings value SECURE_PROXY_SSL_HEADER

### CODE REVIEW

File: app/config/logging-cfg-local.yml [hunk 1]
```diff
@@ -20,10 +20,6 @@ loggers:
     level: DEBUG
     handlers:
       - console
-  default:
-    level: DEBUG
-    handlers:
-      - console
```
Comment (ltshb) [inline app/config/logging-cfg-local.yml:26]: Just for curiosity why did you remove this ?

**Review by ltshb — APPROVED** [review]
:+1: 
Just a general comment about git commit message, as good practice I would suggest to squash both commit and add a proper commit message that explain why these changes were needed. This way when looking at the repo we don't have to search in the PR to understand these changes. So also add the jira ticket in commit. This helps a lot when doing git blame on the code to know why lline of code is as it is.

File: app/config/logging-cfg-local.yml (hunk 1 above)
Comment (boecklic) [reply app/config/logging-cfg-local.yml:26]: I added this some time ago but I think it's not used (since we use `.getLogger(__name__)` everywhere and not a common default logger)
